### PR TITLE
Remove invalid inputs

### PR DESCRIPTION
**Pull request by jonjohnsonjr**
Some of these probably need changes in melange, but I'm going to drop them here first and we can add them back by fixing the melange pipelines to do what the author expected.

### CODE REVIEW

File: feh.yaml [hunk 1]
```diff
@@ -34,8 +34,6 @@ pipeline:
       opts: magic=1
 
   - uses: autoconf/make-install
-    with:
```
Comment (jonjohnsonjr) [inline feh.yaml:37]: FYI @EyeCantCU

File: dropbear.yaml [hunk 2]
```diff
@@ -35,9 +35,6 @@ pipeline:
         PROGRAMS="dropbear dropbearkey dbclient dropbearconvert scp"
 
   - uses: autoconf/make-install
-    with:
-      opts: |
-        PROGRAMS="dropbear dropbearkey dbclient dropbearconvert scp"
```
Comment (jonjohnsonjr) [inline dropbear.yaml:40]: FYI @joshrwolf

File: openjpeg.yaml [hunk 3]
```diff
@@ -30,12 +30,6 @@ pipeline:
   #   with:
   #     patches: fix-cmakelists.patch
   - uses: cmake/configure
-    with:
-      opt: |
-        -G Ninja \
-        -DCMAKE_BUILD_TYPE=MinSizeRel \
-        -DCMAKE_INSTALL_PREFIX=/usr \
-        -DOPENJPEG_INSTALL_LIB_DIR=lib
```
Comment (jonjohnsonjr) [inline openjpeg.yaml:38]: FYI @developer-guy

File: suricata.yaml [hunk 4]
```diff
@@ -68,9 +68,6 @@ pipeline:
         --localstatedir=/var/
 
   - uses: autoconf/make-install
-    with:
-      opts: |
-        DESTDIR=${{targets.destdir}} install-full
```
Comment (jonjohnsonjr) [inline suricata.yaml:73]: FYI @jamonation

File: wireguard-tools.yaml [hunk 5]
```diff
@@ -31,10 +31,6 @@ pipeline:
   - uses: autoconf/make-install
     with:
       dir: src
-      opts: |
-        WITH_BASHCOMPLETION=no \
-        WITH_WGQUICK=no \
-        WITH_SYSTEMDUNITS=no
```
Comment (jonjohnsonjr) [inline wireguard-tools.yaml:37]: FYI @joshrwolf

**Review by imjasonh — DISMISSED** [review]

**Review by imjasonh — DISMISSED** [review]

File: dropbear.yaml (hunk 2 above)
Comment (xnox) [reply dropbear.yaml:40]: I want to fix this the other way around. And change make install to support args just like make pipeline does.

File: feh.yaml [hunk 6]
```diff
@@ -33,9 +33,13 @@ pipeline:
...
-  - uses: autoconf/make-install
-    with:
-      opts: bin_dir=${{targets.destdir}}/usr/bin \ man_dir=${{targets.destdir}}/usr/share/man \ doc_dir=${{targets.destdir}}/usr/share/doc/${{package.name}} \ example_dir=${{targets.destdir}}/usr/share/doc/${{package.name}}/examples
+  - runs: |
+      make \
+        bin_dir=${{targets.destdir}}/usr/bin \
+        man_dir=${{targets.destdir}}/usr/share/man \
+        doc_dir=${{targets.destdir}}/usr/share/doc/${{package.name}} \
+        example_dir=${{targets.destdir}}/usr/share/doc/${{package.name}}/examples \
+        install
```
Comment (xnox) [inline feh.yaml:42]: Make install pipeline should support args like make pipeline does.

File: shadowsocks-rust.yaml [hunk 7]
```diff
@@ -1,6 +1,6 @@
 package:
   name: shadowsocks-rust
-  version: 1.20.0
+  version: 1.20.1
```
Comment (xnox) [inline shadowsocks-rust.yaml:3]: Is this related / intentional?

**Review by xnox — CHANGES_REQUESTED** [review]
I'd leave make-install as it is, and actually make args supported and rebuild these packages again (epoch bump) once the change lands in melange.

File: shadowsocks-rust.yaml (hunk 7 above)
Comment (jonjohnsonjr) [reply shadowsocks-rust.yaml:3]: I wanted to get CI passing, the check update thing was mad at me and I don't have superpowers to merge.

Comment (xnox) [reply shadowsocks-rust.yaml:3]: @jonjohnsonjr update check is not required for merges, one can merge things as is. Likely can bump this separately, but it's also ok to do it here.

**Comment by jonjohnsonjr** [comment]
Closing in favor of https://github.com/wolfi-dev/os/pull/22796 and https://github.com/chainguard-dev/melange/pull/1330
